zsl/interface/celery/worker.py

```diff
@@ -0,0 +1,108 @@
+"""
+:mod:`zsl.interface.celery.worker`
+----------------------------------
+
+Implementation of celery workers.
+
+.. moduleauthor:: Peter Morihladko
+"""
+from __future__ import unicode_literals
+
+import sys
+
+from celery import Celery, shared_task
+from injector import Module, singleton, provides
+
+from zsl import Config
+from zsl.utils.injection_helper import inject
+from zsl.interface.task_queue import TaskQueueWorker
+from zsl.task.job_context import Job
+
+
+class CeleryTaskQueueWorkerBase(TaskQueueWorker):
```

Comment:
I don't know it is the best name, maybe we can shorten to CeleryWorker, but I wanted to denote this is not a complete implementation.